src/rtk/features/auth/authSlice.js

```diff
@@ -0,0 +1,53 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import authenticateUser from "../../../utils/authUtils";
+
+const storedAuthUser = JSON.parse(localStorage.getItem("authUser"));
+
+const initialState = {
+  loading: false,
+  authUser: storedAuthUser || null,
```

Comment:
if there's no `authUser` in localStorage, won't it by default return null/undefined?

Reply:
Yes bhai.

Reply:
Then you can just do this `authUser: storedAuthUser`

Reply:
Okay @ray-wtag  bhai.